tests/framework/resourcemanager.go

```diff
@@ -408,3 +408,40 @@ func (rm *ResourceManager) GetClusterInfo() (ClusterInfo, error) {
 
 	return *ci, nil
 }
+
+// GetNGFPodNames returns the name(s) of the NGF Pod(s).
+func GetNGFPodNames(
```

Comment:
Should we retry this on failure?

Reply:
I don't necessarily think the function itself should have a retry, it should be up to the caller. Similar to a k8sClient.Get call.

Currently, both times this is called, it happens after we use `helm <cmd> --wait`, which should only return when Pods are ready.